scalableminds/webknossos
# Avoid scroll bars caused by context menu positioning

CSS for the win.

Review with "Hide Whitespace" for minimum code changes and maximum pleasure :)

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Test using firefox, chrome
- Open the context menu next to the right or bottom edge. The menu should no longer stick out of the viewport and shouldn't cause any scrollbars to appear.
- The menu items and copy buttons should still work.

### Issues:
- fixes #5383 

------
(Please delete unneeded items, merge only when none are left open)
